git_template/hooks/confirm-changes

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+rebasing_file="$PWD"/.git/rebase_in_progress
+
+if [ -f $rebasing_file ]; then
+  sha_pre_rebase=$(cat $rebasing_file)
+
+  if [ "$(git diff $sha_pre_rebase)" ]; then
+    echo "Your rebase resulted in code changes"
+    git diff $sha_pre_rebase
+
+    while true; do
+      echo  "Do you want to keep the changes? [Y/n] "
+      read -r answer < /dev/tty
+      echo ""
+
+      case $answer in
+        [Yy])
+          echo "Cool! All changes kept"
```

Comment:
A bit noisy for doing nothing. I feel like I'd understand that if I interactively answered "y" do "do you want to keep these changes", the result is that my changes would be kept.